Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 51;
```

Comment:
This should probably be removed from the final patch.

Reply:
I guess, but why. Xcode did it, and someone else’s Xcode will undo it.

Reply:
That's true; but I think it means the project file won't be able to be opened on older versions of Xcode. I know I've been asked to roll out this change in the past as well.

Reply:
That’s backwards. My Xcode is changing the version back to an older version. The thing you were asked to revert was a roll _forward_. Seems super-unlikely that an _older_ `objectVersion` would prevent the file from being opened on _older_ versions of Xcode.

Reply:
You're right! My mistake; I was pattern matching with the +-'s reversed.